hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/utils/ConversionUtils.java

```diff
@@ -439,6 +497,10 @@ private static boolean isLongZeroAddress(final byte[] explicit) {
         return true;
     }
 
+    private static byte[] explicitFromHeadlong(@NonNull final com.esaulpaugh.headlong.abi.Address address) {
+        return unhex(address.toString().substring(2));
+    }
+
```

Comment:
As long as you're in this file consider annotating following line with "too many arguments" to indicate _which_ sonar warning is being suppressed

Reply:
Added comment.